docs/md/modules-covid.md

```diff
@@ -0,0 +1,166 @@
+
+1. The caller needs to create a workload and specify the source, sink and executor. The source must be either a snapshot or a list of snapshots. If the former, then the type is `TDR Snapshots`. If the latter, than it is of type `TDRSnapshotListSource`. The executor must be of type `TerraExecutor`, and the sink must be of type `TerraWorkspaceSink`. If everything else passes verification, then the caller will receive a response which will contain the `uuid` of the workload.
+   
+2. Next, the caller needs to "start" the newly created workload, which will begin the analysis. 
+   
+3. WFL can, in addition, stop watching a workflow using the `stop` endpoint. This will not cancel analysis, but WFL will stop managing the workload.
+
+To give more information, here are some example inputs to the above endpoints:
+
+**GET /api/v1/workload**
+
+==="Sample Request"
+
+    ```
+    curl 'http://localhost:8080/api/v1/workload' \
+        -H 'Authorization: Bearer '$(gcloud auth print-access-token) \
+        -H 'Accept: application/json'
+    ```
+
+**GET /api/v1/workload/{uuid}/workflows**
+
+=== "Sample Request"
+
+    ```
+    curl -X GET 'http://localhost:8080/api/v1/workload/ed6e8637-5bae-4602-8b7f-f5a1bbfd2406/workflows' 
+         -H 'Authorization: Bearer '$(gcloud auth print-access-token)      
+         -H 'Accept: application/json'
+    ```
+
+**POST /api/v1/create**
+
+=== "Sample Request"
+
+    ```
+    curl -X "POST" "http://localhost:8080/api/v1/create" \
+         -H 'Authorization: Bearer '$(gcloud auth print-access-token) \
+         -H 'Accept: application/json' \
+         -H 'Content-Type: application/json' \
+         -d '{
+                "watchers": [
+                    "tester@123.com"
+                ],
+                "labels": [
+                    "hornet:test"
+                ],
+                "project": "wfl-dev/CDC_Viral_Sequencing _ranthony_bashing_copy",
+                "source": {
+                    "name": "Terra DataRepo",
+                    "dataset": "4bb51d98-b4aa-4c72-b76a-1a96a2ee3057",
+                    "table": "flowcells",
+                    "column": "last_modified_date",
+                    "snapshotReaders": [
+                        "workflow-launcher-dev@firecloud.org"
+                    ]
+                },
+                "executor": {
+                    "workspace": "wfl-dev/CDC_Viral_Sequencing _ranthony_bashing_copy",
+                    "methodConfiguration": "wfl-dev/sarscov2_illumina_full",
+                    "methodConfigurationVersion": 2,
+                    "fromSource": "importSnapshot",
+                    "name": "Terra"
+                },
+                "sink": {
+                    "name": "Terra Workspace",
+                    "workspace": "wfl-dev/CDC_Viral_Sequencing _ranthony_bashing_copy",
+                    "entityType": "flowcell",
+                    "identifier": "run_id",
+                    "fromOutputs": {
```

Comment:
Second only to naming things, I find that constructing representative examples to be one of the harder problems we solve :) can't be too vague but also don't want to get bogged down by details that don't aide in our communication.

With that said, I think readability improves if we truncate this map while representing that it's being truncated.  Right now it's 50 lines of identity mappings.  I am happy to be overruled.  Ex.

```
"fromOutputs": {
  "submission_xml" : "submission_xml",
  "assembled_ids" : "assembled_ids",
  "num_failed_assembly" : "num_failed_assembly",
  … 
}
```

Reply:
I like it - done :)